flasksrc/schema.sql

```diff
@@ -22,3 +26,31 @@ CREATE TABLE toynet_value_inspirations (
   quote TEXT NOT NULL,
   FOREIGN KEY (value_id) REFERENCES toynet_value (id)
 );
+
+
+-- quizzes submodule
+
+CREATE TABLE toynet_quizzes (
+  quiz_id INTEGER PRIMARY KEY,
+  created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE toynet_quiz_questions (
```

Comment:
I actually think you can combine `toynet_quiz_questions` with `toynet_quizzes` since they are 1-to-1 what do you think?

Reply:
I think this is a reasonable design choice, because we do not expect to have future relations to the `toynet_quiz_questions` table and this will simplify the overall database. Thoughts John?

Reply:
I already made the change to combine the two tables.